benchmark/heuristic_cache.cpp

```diff
@@ -76,8 +76,8 @@ static auto getLayerBackwardNormRuntime(
   at::Tensor aten_input = at::randn(shape, options);
   at::Tensor aten_weight = at::randn(norm_shape, options);
   at::Tensor aten_bias = at::randn(norm_shape, options);
-  auto at_weight = std::optional<at::Tensor>(aten_weight);
-  auto at_bias = std::optional<at::Tensor>(aten_bias);
+  auto at_weight = c10::optional<at::Tensor>(aten_weight);
+  auto at_bias = c10::optional<at::Tensor>(aten_bias);
```

Comment:
Do we need to wrap things explicitly here? Can't we just use `aten_weight`/`aten_bias` in places of `at_weight`/`at_bias` below?

Reply:
It also works! changed.